pkgs/applications/version-management/josh/default.nix

```diff
@@ -17,17 +17,18 @@ in
 
 rustPlatform.buildRustPackage rec {
   pname = "josh";
-  version = "23.12.04";
-  JOSH_VERSION = "r${version}";
```

Comment:
What did this do?

Reply:
Should be restored: https://github.com/josh-project/josh/blob/f0bc12c22cb4f51cefbdd9d4171c8de956bf1f69/josh-core/src/lib.rs#L91

I'll add a comment explaining this is actually used.

Reply:
Done